generators/app/index.js

```diff
@@ -163,7 +163,7 @@ var BackboneGenerator = yeoman.generators.Base.extend({
         this.destinationPath('.jshintrc'),
         {
           appName: this.appname,
-          appSlugName: paramCase(this.appname),
+          appSlugName: pascalCase(this.appname),
```

Comment:
the name `appSlugName` was just used because it referred to a [different module](https://www.npmjs.com/package/to-slug-case) we were using here at first, now that you're on it, it would be actually nice to replace it to something that makes more sense, maybe `appPascalCaseName` ?

we need to change both here and in the `jshintrc` template file :blush: